charts/kubernetes-sidecar-injector/templates/mutatingwebhook.yaml

```diff
@@ -152,3 +152,19 @@ spec:
         - name: {{ .Release.Name }}-certs
           secret:
             secretName: {{ include "certs.secret.name" . }}
+---
+{{- if .Values.pdb.enabled }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "common.labels" . | indent 4 }}
+spec:
+  minAvailable: {{ .Values.pdb.minAvailable }}
+  maxUnavailable: {{ .Values.pdb.maxUnavailable }}
+  selector:
+    matchLabels:
+      {{- include "common.labels" . | indent 6 }}
+{{- end }}
```

Comment:
This should probably be in it's own file.

Reply:
I was thinking of that but since everything including deployment and others are in the file I thought I would add it here

Reply:
they are together only to fix the certificate refresh between the webhook and the deployment pods.

It is actually fixed and separated in this PR, but I haven't had time to fully test it yet.
https://github.com/ExpediaGroup/kubernetes-sidecar-injector/pull/68